Formula/quarantinecli.rb

```diff
@@ -0,0 +1,21 @@
+class Quarantinecli < Formula
+  desc "CLI Utility to quarantine and manage quarantined files on macOS"
```

Comment:
```suggestion
  desc "Quarantine and manage quarantined files on macOS"
```
We probably don't need to say it's a `CLI Utility`. It's already in the name.